tests/static/all_question_types.xml

```diff
@@ -204,13 +204,14 @@
 					<group>
 						<repeat nodeset="/data/question22/question23/question41" />
 					</group>
-					<upload ref="/data/question22/question23/question24" mediatype="image/*" jr:imageDimensionScaledMax="250px"/>
-					<upload ref="/data/question22/question23/question25" mediatype="audio/*"/>
-					<upload ref="/data/question22/question23/question26" mediatype="video/*"/>
-					<input ref="/data/question22/question23/question27"/>
-					<secret ref="/data/question22/question23/question28"/>
-					<upload ref="/data/question22/question23/question29" mediatype="image/*" appearance="signature"/>
-					<input ref="/data/question22/question23/question7" appearance="intent:question7"/>
+					<upload ref="/data/question22/question23/question24" mediatype="image/*" jr:imageDimensionScaledMax="250px" />
+					<upload ref="/data/question22/question23/question25" mediatype="audio/*" />
+					<upload ref="/data/question22/question23/question26" mediatype="video/*" />
+					<input ref="/data/question22/question23/question27" />
+					<secret ref="/data/question22/question23/question28" />
+					<upload ref="/data/question22/question23/question29" mediatype="image/*" appearance="signature" />
+					<input ref="/data/question22/question23/question7" appearance="intent:question7" />
+					<input ref="/data/question22/question23/question35" mediatype="image/*" appearance="micro-image" />
```

Comment:
nit: the addition of question35 got hidden in the changes to add extra space before the closing tag here.
Good to keep those in separate commits so relevant changes don't get missed.

Reply:
Good point. GitHub doesn't make it very obvious which changes are just spacing changes and which are new lines of code. Will keep this in mind next time.